### DISCUSSION TITLE
Esta es mi práctica html

### DISCUSSION THREAD

**Discussion by sergiolopezloya** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#175] [Show and tell]
Para los que quieran darse una idea, esta es mi práctica:

https://pastel.animeplay.us

Aún me falta el tema del inventario pero es lo que alcancé a hacer ayer durante la transmisión, espero les sirva de guía

**Comment by AngelCruzO** [comment]
Solo una pequeña observación, te falto mostrar los precios de los sabores y de los adornos

**Reply by sergiolopezloya** [reply]
Lo hice durante el live del lunes, me falta también la cuestión del inventario, pero gracias por la observación

**Comment by Ingrzs** [comment]
En donde lo montaste?

**Reply by JuanSalvador73** [reply]
También me pregunto lo mismo.

**Reply by sergiolopezloya** [reply]
Tengo un servidor en AWS, me dedico al hosting de sitios web. animeplay.us es una web mia de una app

**Comment by LuceroLuciano** [comment]
Te quedo Increíble!!
Solo te falta agregar la parte de los precios como lo menciona el explorer @AngelCruzO :)

**Reply by sergiolopezloya** [reply]
Gracias, creo que lo haré en este live :P

**Comment by JuanSalvador73** [comment]
Buen trabajo.

**Reply by sergiolopezloya** [reply]
Gracias compañero
